cli/cli/commands/files/download/download.go

```diff
@@ -99,9 +99,9 @@ func run(
 	flags *flags.ParsedFlags,
 	args *args.ParsedArgs,
 ) error {
-	enclaveIdentifier, err := args.GetNonGreedyArg(enclaveIdentifierArgKey)
+	enclaveIdentifier, err := flags.GetString(enclaveIdentifierFlagKey)
```

Comment:
I believe `enclaveIdentifier ` can now be an empty string. What happens in that case?

Reply:
Oh yeah makes sense; I blindly copied from Kurtosis run! Will correct. Enclave identifier should be mandatory. Will correct 👍

Reply:
ACK, that makes more sense. Feel free to re-tag me as reviewer when you push your change, I can approve